app/importers/reading/somerville_mega_reading_importer.rb

```diff
@@ -1,67 +1,90 @@
+# For importing reading data from Google Sheets or a local folder of csvs.
 class SomervilleMegaReadingImporter
-  def initialize(options:)
-    @log = options.fetch(:log)
-    @school_local_ids = options.fetch(:school_scope, [])
-    @file_text = options.fetch(:file_text, nil)
-
-    # TODO add dependencies as instance variables
-    # @student_ids_map = ::StudentIdsMap.new
+  def initialize(school_year, options = {})
+    @school_year = school_year
+    @log = options.fetch(:log, Rails.env.test? ? LogHelper::Redirect.instance.file : STDOUT)
     # @matcher = options.fetch(:matcher, ImportMatcher.new)
-    @syncer = ::RecordSyncer.new(log: @log)
-    reset_counters!
+    # @syncer = ::RecordSyncer.new(log: @log)
+    @fetcher = options.fetch(:fetcher, GoogleSheetsFetcher.new)
   end
 
   def import
-    streaming_csv = read_or_fetch_csv
-    if streaming_csv.nil?
+    streaming_csvs = read_or_fetch_sheet
```

Comment:
This importer class should work on a folder, not just a single sheet.  Check out https://github.com/studentinsights/studentinsights/blob/master/app/importers/transitions/bedford_davis_transition_notes_importer.rb#L43 for a model of how this is set in the env and how to change the call to the fetcher.

Reply:
Gotcha. I had assumed the template sheet reflected the way the data would be entered - all in one sheet. If instead it will be many sheets in a folder, we'll do that.

Reply:
Sounds good!